packages/trader/src/Stores/Modules/Trading/__tests__/trade-store.spec.ts

```diff
@@ -274,6 +274,7 @@ describe('TradeStore', () => {
                     form_name: 'default',
                     trade_type_name: 'Rise/Fall',
                     ...payload,
+                    duration_type: 'minutes',
```

Comment:
why is the duration_type `minutes` here and `m` above?

Reply:
@balakrishna-deriv it's because the logic for conversion of `duration_unit` ('m', 't', etc.) into a non-localized value ('minutes', 'ticks, etc.), which marketing team wants to receive in Rudderstack, is incapsulated inside `sendTradeParamsAnalytics()`.
In other words, when we pass `duration_unit` ('m', 't', etc.) into `sendTradeParamsAnalytics()` as 'duration_type', it's processing it and passing a non-localized value ('minutes', 'ticks, etc.) to `Analytics.track()` :)